packages/tests/functional/basic.ts

```diff
@@ -1,7 +1,8 @@
 describe('Bigcommerce page', () => {
     it('should have the right title', async () => {
+        console.log('Navigating to bigcommerce.com')
```

Comment:
Do we need the logging?

Reply:
This was to make sure it worked, it'll be removed from the hello-world example